docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -805,9 +805,15 @@ If the extension solves a general problem, it is very handy for Quarkus users to
 But this comes with some responsibility for you, keeping it up to date with Quarkus minor releases (every month or so at the moment).
 When in doubt, have a conversation with the community in the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group]. We can make a collective decision.
 
-[NOTE]
-As for now, the process to propose a new extension is not defined yet.
-Your best chance is to present your extension on the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group] and wait for an official invitation to join the Quarkus Platform.
+If you want to contribute with a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
```

Comment:
```suggestion
If you want to contribute a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
```

Reply:
Actually the correct is `contribute to`: https://dictionary.cambridge.org/us/dictionary/english/contribute-to-sth

Reply:
Well it depends what you mean:

- "contribute to a new extension" means the extension already exists and you want to contribute to it
- "contribute a new extension" means adding a new extension as a contribution

Reply:
Ah that makes sense, let me fix that